tuf/scripts/repo.py

```diff
@@ -509,24 +522,20 @@ def add_verification_key(parsed_arguments):
   repository = repo_tool.load_repository(
       os.path.join(parsed_arguments.path, REPO_DIR))
 
+  expire = None
```

Comment:
Does it never expire by default then?

Reply:
if not set it will take the default value (as before this PR)